### PR TITLE
Fix helix:led_test build break

### PR DESCRIPTION
**Pull request by mtei**
#2922 cause helix:led_test build break.

This PullRequest fix build break.
 * copy new quantum/rgblight.[ch] into keyboards/helix/rev2/keymaps/led_test/
 * add new rgblight mode 35 (RGB cyclic) into local rgblight.[ch]

### CODE REVIEW

File: keyboards/helix/rev2/keymaps/led_test/keymap.c [hunk 1]
```diff
@@ -513,6 +513,12 @@ void music_scale_user(void)
 #ifdef SSD1306OLED
 
 void matrix_scan_user(void) {
+    static int scan_count = 0;
```
Comment (drashna) [inline keyboards/helix/rev2/keymaps/led_test/keymap.c]: I'm hoping to have a fix for this.  

Namely a "keyboard_init_" function that is called late enough that it doesn't get overwritten, but for now, this should be perfect.

**Comment by drashna** [comment]
I was wanting to mention the "noeeprom" overhaul to you, but it looks like you've already added it! :)

**Comment by drashna** [comment]
Nice! Thanks!

**Comment by mtei** [comment]
Thank you!

And, I hope this rgblight mode 35 (RGB cyclic) merge into quantum/rgblight.[ch].
What do you think about this?

Can I change quantum/rgblight.[ch] and PullRequest ?

**Comment by drashna** [comment]
If it works well, then yes, by all means, add it to the main rgblight.c file.

Just ... make sure there are keycodes for it too, https://docs.qmk.fm/#/feature_rgblight?id=rgb-lighting-keycodes

And update the docs, as appropriate.

**Pull request merged**